config/conventionalChangelog/config.js

```diff
@@ -49,10 +49,6 @@ var writerOpts = {
             discard = false;
         });
 
-        if (commit.header && commit.header.includes('-next.')) {
```

Comment:
Don't we need to keep this to avoid publishing too many changelogs?

Reply:
Not anymore, since next version commits are now pushed to the `next` branch. The changelog only looks for commits on `master` branch whose commit tree is untouched by the "next" versioning.